Source/AwsCommonRuntimeKit/io/TlsContext.swift

```diff
@@ -26,3 +24,8 @@ public class TlsContext {
         aws_tls_ctx_release(rawValue)
     }
 }
+
+public enum TlsMode {
```

Comment:
Nit: I think it'd be preferable to name this `TLSMode`

Reply:
Thanks, updated.